jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -76,6 +76,15 @@
         trigger-job: 'ci-kubernetes-build'
     # END AWS
 
+    # START CHARTS
+    - kubernetes-e2e-charts-gce: # foxish
+        job-name: ci-kubernetes-e2e-charts-gce
+        timeout: 100
```

Comment:
bootstrap timeout need to be longer than e2e timeout, which is > 180m. 200 would be fine.